addons/mail/static/src/js/chat_window.js

```diff
@@ -73,21 +75,24 @@ return Widget.extend({
         this.unread_msgs = counter;
         this.render_header();
     },
-    update_status: function (status) {
+    update_status: function (status, last_seen_on) {
```

Comment:
missing docstring: please document your functions (in particular, the type of the parameters)